README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
@kevinsawicki well, it's good to know, because was looking at atom and I have implemented the similar code as at atom for my application. Do you know, from which version electron supports auto updates for windows in the same way, as for osx?